### PR TITLE
Adds Files page callout in Query.

### LINKED ISSUE #65: How To: Add website info to query how-to
You can get some of the status information from the files pages of the website. We should mention that in our docs.

### PR DESCRIPTION
**Pull request by johnnymatthews**
Closes #65

### CODE REVIEW

File: docs/how-tos/query.md [hunk 1]
```diff
@@ -96,6 +97,12 @@ What do all those fields mean? Here's a summary:
 
 For more details about the fields in this JSON response, including the format of the `pins` and `deals` objects, see the [JavaScript client library reference][reference-js-status].
 
+:::tip Use the website
```
Comment (jessicaschilling) [inline docs/how-tos/query.md:100]:
```suggestion
:::tip
```
So as not to imply that using the website is the only option.

File: docs/how-tos/query.md [hunk 2]
```diff
@@ -96,6 +97,12 @@ What do all those fields mean? Here's a summary:
...
+The **Files** page on [web3.storage](https://web3.storage) lists some commonly used attributes found using `query()`.
```
Comment (jessicaschilling) [inline docs/how-tos/query.md:101]:
```suggestion
If you're looking for info on files you've uploaded, you can also use the **Files** page on [web3.storage](https://web3.storage) to see the values for some of the more commonly-used attributes returned by `query()`, namely `created`, `cid`, `dagSize`, and the `status` and `deals` objects of `pins`.
```
Clarify that this only works for files _you've_ uploaded, and add the list of values.

File: docs/how-tos/query.md [hunk 3]
```diff
@@ -96,6 +97,12 @@ What do all those fields mean? Here's a summary:
...
+
+![Web3.Storage showing the Files page](./images/files-page-in-web3-storage.png)
```
Comment (jessicaschilling) [inline docs/how-tos/query.md:103]:
```suggestion
```
Suggest we omit this for now for two reasons: 
- screenshots in tip boxes are confusing visually
- this screenshot will change in the coming days

**If you agree, please remove the image from the directory as well.**

**Review by jessicaschilling — CHANGES_REQUESTED** [review]
Thanks, @johnnymatthews - added a few small comments. Thank you!

**Comment by johnnymatthews** [comment]
Merged your suggestions and removed the image @jessicaschilling.

**Comment by filecorgi** [comment]
- Image optimization came back clean!
- Vuepress build was successful!
### `docs/how-tos/query.md`:
- The following link failed to connect:
  https://web3.storage
- Possibly missing preposition found.
  ```
  ... files. When you store a file with Web3.Storage, you receive a content ident...
                                             ^^^^^^^
  ```
- Possibly missing comma found.
  ```
  ...re's how to include it in your project: IMPORTANT Remember to check the retu...
                                             ^^^^^^^^^
  ```
- Possibly missing preposition found.
  ```
  ... is valid and has been uploaded to Web3.Storage, the status method will retu...
                                             ^^^^^^^
  ```
- Possible agreement error.
  ```
  ...an the total size of the files on disk. pins contains an array of objects de...
                                             ^^^^^^^^^^^^^
  ```
- Possible agreement error.
  ```
  ... fast retrieval using the IPFS network. deals contains an array of objects d...
                                             ^^^^^^^^^^^^^^
  ```
- Although a hyphen is possible, it is not necessary in a compound modifier in which the first word is an adverb that ends in ‘ly’.
  ```
  ... to see the values for some of the more commonly-used attributes returned by...
                                             ^^^^^^^^^^^^^
  ```
- The verb “check out” can sound colloquial in this context. Consider using a more formal alternative.
  ```
  ...h how to store data using Web3.Storage, check out the storage how-to guide f...
                                             ^^^^^^^^^
  ```

**Comment by filecorgi** [comment]
- Image optimization came back clean!
- Vuepress build was successful!
### `docs/how-tos/query.md`:
- The following link failed to connect:
  https://web3.storage
- Possibly missing preposition found.
  ```
  ... files. When you store a file with Web3.Storage, you receive a content ident...
                                             ^^^^^^^
  ```
- Possibly missing comma found.
  ```
  ...re's how to include it in your project: IMPORTANT Remember to check the retu...
                                             ^^^^^^^^^
  ```
- Possibly missing preposition found.
  ```
  ... is valid and has been uploaded to Web3.Storage, the status method will retu...
                                             ^^^^^^^
  ```
- Possible agreement error.
  ```
  ...an the total size of the files on disk. pins contains an array of objects de...
                                             ^^^^^^^^^^^^^
  ```
- Possible agreement error.
  ```
  ... fast retrieval using the IPFS network. deals contains an array of objects d...
                                             ^^^^^^^^^^^^^^
  ```
- Although a hyphen is possible, it is not necessary in a compound modifier in which the first word is an adverb that ends in ‘ly’.
  ```
  ... to see the values for some of the more commonly-used attributes returned by...
                                             ^^^^^^^^^^^^^
  ```
- The verb “check out” can sound colloquial in this context. Consider using a more formal alternative.
  ```
  ...h how to store data using Web3.Storage, check out the storage how-to guide f...
                                             ^^^^^^^^^
  ```

**Review by jessicaschilling — APPROVED** [review]
LGTM - @atopal can you please merge if you're OK with this?

**Review by atopal — APPROVED** [review]
Thank you!

**Pull request merged**